app/src/test/java/by/academy/jc/ht/zhabko/MayanCalendarTest.java

```diff
@@ -0,0 +1,54 @@
+package by.academy.jc.ht.zhabko;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+public class MayanCalendarTest {
```

Comment:
Please rework `MayanCalendarTest` in the way that it will not only show what happens with dates but also asserts that expected values are the same as actual ones.